docs/environments/atari/amidar.md

```diff
@@ -53,6 +53,7 @@ See variants section for the type of observation used by each environment id by
 You score points by traversing new parts of the grid. Coloring an entire box in the maze or catching chickens gives extra points.
 For a more detailed documentation, see [the AtariAge page](https://atariage.com/manual_html_page.php?SoftwareID=817).
```

Comment:
It would be great for all of the games to be consistent such that we either had the documentation website line in the paragraph or on a new line. Could you see if there is an easy way of making this consistent given we have a script that should do most of the structuring for us

If this is not easily possible, then don't worry

Reply:
by majority rule: I made them all start on a new line :)